data-prepper-plugins/rds-source/src/main/java/org/opensearch/dataprepper/plugins/source/rds/export/DataFileScheduler.java

```diff
@@ -127,7 +125,7 @@ private void processDataFilePartition(DataFilePartition dataFilePartition) {
                 updateLoadStatus(dataFilePartition.getExportTaskId(), DEFAULT_UPDATE_LOAD_STATUS_TIMEOUT);
                 sourceCoordinator.completePartition(dataFilePartition);
             } else {
-                LOG.error("There was an exception while processing an S3 data file: {}", ex);
+                LOG.error("There was an exception while processing an S3 data file", (Throwable) ex);
```

Comment:
You shouldn't need this cast.

Reply:
Added because of a warning in my IDE. But looks like it's not necessary.